package.json

```diff
@@ -6,13 +6,14 @@
   "directories": {
     "doc": "docs"
   },
-  "assetsPath" : "showcase/public/assets/",
+  "assetsPath": "showcase/public/assets/",
   "scripts": {
     "analyse": "elm-analyse",
     "docs": "elm make --docs=dist/docs.json",
     "build": "parcel build showcase/public/index.html --public-url ./",
     "showcase": "parcel showcase/public/index.html",
-    "test": "elm-test"
+    "test": "elm-test",
+    "i18n": "node -e 'require(\"./node_modules/parcel-plugin-elm-i18n/index\")()' && elm-format --yes src/I18n"
```

Comment:
That's a bit hackish I would say, you can create runnable commands in the `parcel-plugin-elm-i18n` to make it runnable as something like: `npx convert-i18n-to-elm` or something like that.

Also, I think this elm-format should be handled by the library and not here 🤔